### DISCUSSION TITLE
Splitter: panel is not draggable by touch to resize on the android tablet.

### DISCUSSION THREAD

**Discussion by ks-pang-hao** [primefaces/community#2107] [PrimeFaces] [answered]
Hi, I just found out that the splitter panel is not draggable to resize on my tablet, even in Chrome development mode on my dev PC. We can reproduce the problem by visiting the official demo page in Chrome dev mode and selecting the iPad simulator. 
The problem on my tablet occurs only when I use the touchscreen, but it works correctly when I connect and control it with a mouse on the same device. Please help check it out. Thanks.

Primefaces version : 14.0.0
Table: Lenovo P11 android 13
Chrome pc latest version

**Comment by melloware** [comment]
I just tested it on my real iPhone and touch mode is working.  

When I try it with Chrome Debug tools I see this jQuery error saying its bad.

![image](https://github.com/primefaces/community/assets/4399574/464f570d-7a37-4ad2-baba-01a0329713d8)

**Reply by melloware** [reply]
I have to find a real android device to test with.

**Reply by melloware** [reply]
Can you please open a GitHub issue on PrimeFaces please.

**Reply by melloware** [reply, marked as answer]
I opened the ticket here: https://github.com/primefaces/primefaces/issues/12106

**Comment by ks-pang-hao** [comment]
I saw the issue has been fixed, great job, thanks!

**Reply by melloware** [reply]
yep will be in 14.0.1

**Reply by melloware** [reply]
14.0.1 is in Maven Central if you want to try it.

**Reply by ks-pang-hao** [reply]
I just updated to the 14.0.1, and  it works pretty well on my android tablet. thanks agains!
